ansible/bootstrap.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Bootstrap node servers
+  hosts: all
+  remote_user: master
+  become: yes
+
+  vars:
+    bootstrap_version: master
+    vault_role: ae-node
+    package: "https://s3.eu-central-1.amazonaws.com/aeternity-node-builds/aeternity-latest-ubuntu-x86_64.tar.gz"
+
+  tasks:
+    - name: Clone infrastructure repo
+      git:
+        repo:  https://github.com/aeternity/infrastructure.git
+        dest: /infrastructure
+        version: "{{ bootstrap_version }}"
+        depth: 1
+        force: yes
+
+    - name: Run bootstrap.sh
+      shell: |
+        /infrastructure/scripts/bootstrap.sh \
+         --vault_addr={{ vault_addr }} \
+         --vault_role={{ vault_role }} \
+         --env={{ env }} \
+         --epoch_package={{ package }} \
+         >> /tmp/user-data.log 2>&1
+      async: 900
```

Comment:
is this 15 minutes? Isn't it too much, what's the runtime on new instance ?

Reply:
AFAIR it's less than 5 mins, i thought it's not good to fail because of slow download. I'll lower it